src/Rules/StarterWeb/AI/IndividualAuth/appsettings.json

```diff
@@ -2,11 +2,7 @@
   "ApplicationInsights": {
     "InstrumentationKey": ""
   },
-  "Data": {
-    "DefaultConnection": {
-      "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true"
-    }
-  },
+  "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-$safeprojectname$-$guid2$;Trusted_Connection=True;MultipleActiveResultSets=true",
```

Comment:
Why not simply call the key ConnectionString instead of DefaultConnection? What does "default" even mean in this context?